tests/foreman/cli/test_fact.py

```diff
@@ -24,19 +24,24 @@
 
 pytestmark = [pytest.mark.tier1]
 
-
+@pytest.mark.skip_if_open('BZ:2161294')
+@pytest.mark.skip
```

Comment:
```suggestion
@pytest.mark.skip_if_open('BZ:2161294')
```
If I'm assuming correctly, the test should pass once the BZ is fixed. Therefore, only need the skip if open, since we want the test to run once the BZ is closed to make sure the issue is resolved.